src/Compilers/Core/Portable/Operations/Statement.cs

```diff
@@ -1,13 +1,19 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Collections.Immutable;
+
 namespace Microsoft.CodeAnalysis.Semantics
 {
     internal partial class VariableDeclaration : IVariableDeclaration
     {
         public VariableDeclaration(ILocalSymbol variable, IOperation initialValue, SyntaxNode syntax) :
-            this(variable,
+            this(ImmutableArray.Create(variable), initialValue, syntax)
+        {
+        }
+        public VariableDeclaration(ImmutableArray<ILocalSymbol> variables, IOperation initialValue, SyntaxNode syntax) :
+            this(variables,
                 initialValue,
-                variable == null || (initialValue != null && initialValue.IsInvalid),
+                variables == null || (initialValue != null && initialValue.IsInvalid),
```

Comment:
>variables == null [](start = 16, length = 17)

```ImmutableArray<ILocalSymbol>``` is a structure. What does it mean to compare it with null? What do we want to check for here?